cypress/integration/login-logout/login-logout.spec.js

```diff
@@ -19,14 +19,8 @@ describe('User Login Logout', () => {
       cy.get('input[name=github_url]').type('https://github.com/dearshrewdwit')
       cy.get('#user-submit-button').click()
 
-      cy.get('#user-login-link').click()
```

Comment:
This test is not valid because it doesn't test whether you can actually login anymore!

You should have updated the /integration/register tests to check whether you get auto-logged in after successful registration.

I suggest redoing this specific test to actually log in a user; perhaps this test does this:

1) registers a new user
2) clicks logout (due to auto login)
3) then clicks the button to user-login-link
4) types up the user's details
5) clicks submit/login

Reply:
The test "can log out after login" does this flow, but we should probably rename the tests to reflect what they are doing

Reply:
is this fine if i change the test name to this
'user is registered and automatically logged in'

Reply:
Before :     it('can log out after login', () => {
                   cy.get('#user-registration-link').click()

After :     it('user is registered and automatically logged in', () => {
                   cy.get('#user-registration-link').click()